### ISSUE TITLE
Conflicts to the Network Structure

### ISSUE THREAD

**Issue by quocdat32461997** [XifengGuo/DEC-keras#20]
Hi all, 

Thanks for sharing a great Model for Deep Clustering. I aim to leverge your excellent work to cluster images for my project. 

Baed on your paper, the Network Structure is a basis of Stacked Autoencoder which is composed of 2 pairs of Dropout-Dense. However, the implementation does not follow the network structure. Is there any reason behind?

**Comment by shashidhar100** [comment]
Hi,
In the original paper he has used stacked autoencoder and he has trained every layer before stacking it.
and accuracy of your model according to the result table is best then in that  original paper,
does your dec implementation is better than original paper without using stacked autoencoder

**Comment by XifengGuo** [comment]
@quocdat32461997 @shashidhar100 

As I said in #14, the pretraining strategy and weight initialization method are different with the original paper.
I found the layer-wise pretraining is tedious, so I tried the end-to-end training, and it turned out performs better after I changed the weight initialization method and optimization parameters.
